### ISSUE TITLE
Automate OBS Settings

### ISSUE THREAD

**Issue by jasondl13** [JRoy/InstagramLive-PHP#59]
It's possible to modify the OBS profile settings in "%appdata%\obs-studio\basic\profiles\Untitled" to set the size of the live stream and stream url/key. Furthermore, it's possible to use PHP to automatically kill and load OBS to automate this further.

I already coded some of this for a personal project of mine so if you want me to help out I'd be glad, just hit me up.

**Comment by JRoy** [comment]
I could look into OBS Automation with a flag or something. If you would like to share some of the code here that would be nice to look off of as well.

**Comment by jasondl13** [comment]
[instalive.zip](https://github.com/JRoy/InstagramLive-PHP/files/2716725/instalive.zip)
Here's the wrote I wrote/modified. obs.php contains the functions for opening OBS, closing OBS, and updating stream key. goLive.php starting at line 195 is a modified version that allows OBS to load automatically.
Obviously when I wrote this I was just writing it for my personal project so it'll require some adaptation to make it more dynamic.
